src/MizarWidgets/include/MizarWidgets/SamplingWithFrameTrackInputWidget.h

```diff
@@ -28,10 +28,14 @@ namespace Ui {
 class SamplingWithFrameTrackInputWidget;
 }
 
+Q_DECLARE_METATYPE(::orbit_mizar_base::TID);
```

Comment:
Should this maybe be in ThreadId.h ?

Reply:
Theoretically ThreadId.h could later be included from a library that doesn't have qt dependency. So I'm reluctant to do so.

Reply:
Yes, I guessed that was the reason behind it. SGTM. Resolving this